schema/schema.gql

```diff
@@ -511,14 +511,21 @@ type Mutation {
     """The account that will be approved to operate the collection."""
     operator: String!
 
+<<<<<<< master
     """enjin-platform::args.signingAccount"""
     signingAccount: String
 
+=======
+>>>>>>> Adds signingPayload
     """Simulates a transaction without broadcasting it to the network."""
     simulate: Boolean = false
 
     """Skip all validation rules, use with caution. Defaults to false."""
+<<<<<<< master
```

Comment:
Seems like conflicts were not resolved correctly

Reply:
This file is not used at all anyway thank you. I will open another PR to fix anyway.